plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/DeployPreferencesDialog.java

```diff
@@ -157,11 +161,6 @@ private IStatus validateAppEngineJavaComponents()  {
         Status status = new Status(IStatus.ERROR,
             "com.google.cloud.tools.eclipse.appengine.deploy.ui", detailMessage);
         return status;
-    } catch (AppEngineException ex) {
```

Comment:
`CloudSdkOutOfDateException` is being caught right above. (Not shown unless you expand this GitHub diff.)